docs/onboarding/installation.md

```diff
@@ -9,18 +9,15 @@ hide_table_of_contents: false
 import Terminal from '../../src/theme/components/Terminal/Terminal'
 import Ironfish from '../../src/theme/components/Terminal/Ironfish'
 
-## macOS
-
-### Homebrew
-
-> **Note:** Our Homebrew tap does not yet support **M1/arm64**. However, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker) with x86_64 support.
-
-Start by adding our Homebrew tap and installing Iron Fish.
+## Recommended Installation
+We recommend installing Iron Fish through NPM.
```

Comment:
Maybe valuable to link to https://nodejs.org/en/download/ ?

Reply:
There's a link to Node in the note right under it, but it may be prudent to include one there as well.